Docs/Book/Install.md

```diff
@@ -84,12 +112,15 @@ RDBASE=$RDBASE DYLD_FALLBACK_LIBRARY_PATH="$RDBASE/lib:$PYROOT/lib" PYTHONPATH=$
 This is required due to the [System Integrity Protection SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection)
 introduced in more recent macOS versions.
 
-#### Linux x86_64: Python 3 environment
+### Linux x86_64: Python 3 environment
 
 The following commands will create a development environment for Linux x86_64 and Python 3.
+Two modes are possible:
+- **Mode A:** Build with `RDK_INSTALL_INTREE=OFF` and `RDK_BUILD_PYTHON_WRAPPERS=ON`. This mode allows you to interactively modify the C++ code and test the changes via Python wrappers.
+- **Mode B:** Build with `RDK_INSTALL_INTREE=ON` and `RDK_BUILD_CPP_TESTS=ON`. This mode allows you to run the C++ tests and Python tests.
```

Comment:
I disagree with this.
Whether or not the C++ tests are built and run has no connection to whether or no the python wrappers are built and tested.
Similarly, both can be built/tested with either an in-tree or out-of-tree build.

I don't see particular value in enumerating/explaining all possible options in this document... there are just too many combinations possible